packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -95,16 +95,14 @@ Obsoletes: %{?scl_prefix}rubygem-bastion
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.3.2
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 3.8.0
 BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 4.0.0
 BuildRequires: %{?scl_prefix}npm(babel-loader) >= 8.0.0
 BuildRequires: %{?scl_prefix}npm(babel-loader) < 9.0.0
 BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) >= 3.0.0
 BuildRequires: %{?scl_prefix}npm(identity-obj-proxy) < 4.0.0
 # end package.json devDependencies BuildRequires
 # start package.json dependencies BuildRequires
-BuildRequires: %{?scl_prefix}npm(@theforeman/vendor) >= 3.3.2
```

Comment:
This is because of https://github.com/Katello/katello/commit/c628ca1fac906c6429fe1944fca13fadf8241a00. We probably also need to convert `peerDependency`

Reply:
Probably. Can we fix up the PR by hand until `update-requirements` knows what a `peerDependency` is?